infrastructure/data/user.ts

```diff
@@ -4,6 +4,7 @@ import Chance from 'chance';
 const { PrismaClient } = Prisma;
 
 const chance = new Chance();
+
 const prisma = new PrismaClient();
```

Comment:
The committed change was missing a semi-colon. The semi-colon was re-added :100: